### DISCUSSION TITLE
Project Beta Re-ordering of Issues/Items is Painful

### DISCUSSION THREAD

**Discussion by webern** [community/community#17839] [Other Feature Feedback, Questions, & Ideas]
You can only drag one item at a time, not multiple.
There is only a small place that you can grab the item to drag it, where there is a drop down menu triangle (odd).
If you need to scroll while dragging the item, the scroll does not happen automatically based on the drag.
There is no quick way to move an item to the top of the list, you have to drag it all the way there.

**Comment by ronnyroeller** [comment]
Agreed. This makes working with many issuesitems (100+) quite painful.
It's especially confusing that one can select multiple items but then move only one.

**Comment by Mbarnes2022** [comment]
[]()![Uploading c{e12bfea} (ed400268-e505-43e3-b1fd-065b0637efa9 id=0x7f0a03c5 tag=BaseCommentFrAll new builds and different locations throughout Ohio and different home developers all done by precision depfu/update/rubocop-lts-22.0.1 and Barnes All new builds and different locations throughout Ohio and different home developers all done by precision depth/update/rubocop-lts-22.0.1 and Barnes depth/update/rubocop-lts-22.0.1 am I homes in plain City[]() am I homes in plain Statement) …]()[]() Barnes builders only one location

**Comment by JayseMayne** [comment]
normalize-url
